libs/types-common/src/Control/Budget.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 
-module Brig.Budget
```

Comment:
We can't move this outside of brig as it makes assumptions about the brig schema